state-chain/runtime/src/chainflip.rs

```diff
@@ -1021,3 +1022,42 @@ impl cf_traits::MinimumDeposit for MinimumDepositProvider {
 		}
 	}
 }
+
+use sp_runtime::Saturating;
+
+pub struct CfTransactionFeeScaler;
+impl TransactionFeeScaler<RuntimeCall, AccountId, FlipBalance> for CfTransactionFeeScaler {
+	fn call_info(call: &RuntimeCall, caller: &AccountId) -> Option<CallInfoId<AccountId>> {
+		match call {
+			RuntimeCall::LiquidityPools(pallet_cf_pools::Call::set_limit_order {
+				base_asset,
+				..
+			}) |
+			RuntimeCall::LiquidityPools(pallet_cf_pools::Call::update_limit_order {
+				base_asset,
+				..
+			}) |
+			RuntimeCall::LiquidityPools(pallet_cf_pools::Call::set_range_order {
+				base_asset,
+				..
+			}) |
+			RuntimeCall::LiquidityPools(pallet_cf_pools::Call::update_range_order {
+				base_asset,
+				..
+			}) => Some(CallInfoId::Pool(PoolTouched {
```

Comment:
Why not differentiate between limit orders and range orders?

Reply:
Why would we?

Reply:
Because it's valid to update a range order and limit order in the same block (they are different calls) whereas with multiple limit order updates, all but the last are effectively no-ops and just consume block space for no reason.

Reply:
There could be a strategy that uses both range order and limit orders, updating ranges between their limit order ticks to catch volume between ticks. This would be a rational strategy. 
Anyone using range orders as is typically used isn't going to hit rate limiting anyway and so this wouldn't apply.

Reply:
Not really sure what the argument is here. Why would we rate-limit both together? ie. Why should an LP pay more for a limit order if they already placed a range order?